templates/cluster-template-machinepool.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  name: "${CLUSTER_NAME}"
+spec:
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+    kind: AWSCluster
+    name: "${CLUSTER_NAME}"
+  controlPlaneRef:
+    kind: KubeadmControlPlane
```

Comment:
It would be great to also create a `eks-machinepool` flavor.

Reply:
I think this can be done in a follow up issue.

Reply:
sounds good!

Reply:
Created #2029 for the follow up.